README.md

```diff
@@ -46,23 +47,24 @@ A response example is provided below.
 ```
 
 ##Authors by UUID
-`GET /transformers/authors/{uuid}` returns author data of the given uuid. 
+`GET /transformers/author-memberships/{uuid}` returns author membership data of the given uuid. 
 A response example is provided below.
 
 ```
 {
-  "uuid": "daf5fed2-013c-468d-85c4-aee709b8aa53",
-  "identifiers": [
+  "uuid": "e6e8b0e4-4833-11e6-beb8-9e71128cae77",
+  "prefLabel": "Chief Economics Commentator",
+  "personUuid": "6f53299a-799d-49ae-ae9d-7e1f298daef7",
+  "organisationUuid": "dac01f07-4b6d-3615-8532-a56752cc7e5f",
+  "alternativeIdentifiers": {
+    "uuids": [
+      "e6e8b0e4-4833-11e6-beb8-9e71128cae77"
+    ]
+  },
+  "membershipRoles": [
     {
-      "authority": "http://api.ft.com/system/FT-TME",
-      "identifierValue": "Q0ItMDAwMDkwMA==-QXV0auiycw=="
+      "roleuuid": "7ef75a6a-b6bf-4eb7-a1da-03e0acabef1b"
```

Comment:
That's worrying - I was looking at what comes out of the other membership transformer, which looks like this:
"membershipRoles": [
{
"roleUuid": "7752a1f1-360d-3aaf-92d8-448307ce09fb",
"inceptionDate": "1995-01-01T00:00:00.000Z"
}...

I think this may be a bug in membership-rw.